bigbluebutton-html5/imports/ui/components/video-provider/component.jsx

```diff
@@ -45,6 +45,10 @@ const intlClientErrors = defineMessages({
     id: 'app.video.notReadableError',
     description: 'error message When the webcam is being used by other software',
   },
+  iceConnectionStateError: {
+    id: 'app.video.iceConnectionStateError',
```

Comment:
Couldn't we reuse the ICE connectivity message from audio?https://github.com/bigbluebutton/bigbluebutton/blob/fdf0615d04ea44e3dbafd7863a10347e00e93bb6/bigbluebutton-html5/private/locales/en.json#L231

Reply:
I agree. I was referring to the message per se (`ICE negotiation failed`), to be clear, not the error code.

Reply:
So do you know what the code should be? 2007?

Reply:
Increase either the first or second digit for kurento codes. The last two digits can be whatever you want though. I would probably make them sequential so it's easier to add new ones. There should also be some sort of resource in the docs that lists what each of the codes means. I did the write up for the audio codes and it's a useful reference so people don't have to reference the code directly. See, http://docs.bigbluebutton.org/install/install.html#client-webrtc-error-codes

Reply:
We already have some SFU errors: https://github.com/bigbluebutton/bigbluebutton/blob/v2.0.x-release/bigbluebutton-html5/imports/ui/components/video-provider/component.jsx#L50.

Reply:
Ah true. Didn't know those existed.

Reply:
I think client-side WebRTC errors should be 11xx as @capilkey suggested. The SFU errors are server-side errors that are propagated to the clients. Hence the 2xxx vs 1xxx differentiation. TL;DR server-side 2xxx, client-side 1xxx.

Reply:
I like that distinction. Helps determine the source even more.